test/hotspot/jtreg/serviceability/sa/ClhsdbDumpheap.java

```diff
@@ -39,6 +46,9 @@
  */
 
 public class ClhsdbDumpheap {
+    private static final String kHeapDumpFileNameDefault = "heap.bin";
+    private static final String kHeapDumpFileNameGzDefault = "heap.bin.gz";
```

Comment:
I'm not sure about the naming here. Why the `k` prefix? Usually in java static final variables are all upper case with `_` separating the words. So maybe something like HEAPDUMP_FILENAME_DEFAULT